probe-dictionary/explore.js

```diff
@@ -1085,6 +1094,14 @@ function renderProbeStats() {
       .attr("fill", "#000")
       .text("Count of " + constraintText + " probes");
 
+  var columns = [];
```

Comment:
A small thing:
I only now noticed that we already have `var columns` defined on line 1019.
Let's just rename this one to `var legendLabels` or so?